package.json

```diff
@@ -18,24 +18,22 @@
   ],
   "author": "Michael Oakley <michael.oakley@nrel.gov>",
   "devDependencies": {
+    "@babel/preset-env": "^7.12.11",
     "babel-preset-es2015": "^6.3.13",
-    "delete": "^0.3.0",
-    "gulp": "^3.9.0",
-    "gulp-babel": "^6.1.1",
-    "gulp-clean": "^0.3.1",
-    "gulp-concat": "^2.6.0",
-    "gulp-cssmin": "^0.1.7",
-    "gulp-delete-file": "^1.0.2",
-    "gulp-eslint": "^1.1.1",
+    "cssnano": "^4.1.10",
+    "gulp": "^4.0.2",
+    "gulp-babel": "^8.0.0",
+    "gulp-concat": "^2.6.1",
+    "gulp-eslint": "^6.0.0",
     "gulp-gh-pages": "^0.5.4",
-    "gulp-jshint": "^2.0.0",
-    "gulp-livereload": "^3.8.1",
-    "gulp-rename": "^1.2.2",
-    "gulp-sourcemaps": "^1.6.0",
-    "gulp-uglify": "^1.5.1",
-    "gulp-usemin": "^0.3.16",
-    "jshint": "^2.8.0",
-    "run-sequence": "^1.1.5"
+    "gulp-postcss": "^9.0.0",
+    "gulp-rename": "^2.0.0",
+    "gulp-sourcemaps": "^3.0.0",
+    "gulp-uglify": "^3.0.2",
+    "gulp-usemin": "^0.3.30"
```

Comment:
This is the latest version of `gulp-usemin`, but it is deprecated and suggests using webpack or browserify. Assuming we will want to update that, but haven't had a chance to atm.